Buildings/Experimental/NatVentControl/VAV/VAVAirflow.mo

```diff
@@ -0,0 +1,70 @@
+within Buildings.Experimental.NatVentControl.VAV;
+block VAVAirflow
+  "Sets airflow for VAV based on natural ventilation mode and room population"
+parameter Real AreaMinFlo(min=0,
+    final unit="m3/s",
+    final displayUnit="m3/s",
+    final quantity="VolumeFlowRate")=0.1  "Area driven minimum airflow";
+ parameter Real VentMinFlo(min=0,
+    final unit="m3/s",
+    final displayUnit="m3/s",
+    final quantity="VolumeFlowRate")=0.3  "Ventilation minimum airflow";
```

Comment:
Is this the population component of the required breathing zone outdoor airflow setpoint?

Reply:
this was the combined people & area rate